keyboards/cyboard/dactyl/config.h

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2023 Cyboard LLC (@Cyboard-DigitalTailor)
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ */
+
+#pragma once
+
+/* Split keyboard settings */
+#define SERIAL_USART_TX_PIN GP0
+
+/* RGB settings */
+#ifdef RGB_MATRIX_ENABLE
+#    define WS2812_PIO_USE_PIO1 // Force the usage of PIO1 peripheral, by default the WS2812 implementation uses the PIO0 peripheral
+#    define SPLIT_TRANSPORT_MIRROR
+#    define RGB_MATRIX_LED_COUNT 80
+#    define RGB_MATRIX_DEFAULT_MODE RGB_MATRIX_SOLID_REACTIVE // Sets the default mode, if none has been set
+#    define RGB_MATRIX_DEFAULT_HUE 33 // Sets the default hue value, if none has been set
+#    define RGB_MATRIX_DEFAULT_SAT 255 // Sets the default saturation value, if none has been set
+#    define RGB_MATRIX_DEFAULT_SPD 50
+#    define RGB_MATRIX_DEFAULT_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS
+#    define RGB_DISABLE_WHEN_USB_SUSPENDED
+#    define RGB_MATRIX_KEYPRESSES
+#    define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#endif
+
+/* Double-tap reset button parameters */
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_LED GP17
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 1000U
+
+// fix for device detection after cold boot
+// https://github.com/qmk/qmk_firmware/issues/18511#issuecomment-1264604610
+// https://github.com/qmk/qmk_firmware/issues/8990#issuecomment-851055637
+#undef SPLIT_USB_DETECT
+//https://www.reddit.com/r/ErgoMechKeyboards/comments/11qevf9/comment/jc5y8ux/?context=3
```

Comment:
This shouldn't be needed, at all. 

If `USB_VBUS_PIN` is defined, then `SPLIT_USB_DETECT` is not defined normally. 
https://github.com/qmk/qmk_firmware/blob/master/platforms/chibios/chibios_config.h#L18-L20

```suggestion
```

Reply:
Ah, makes sense.  Thanks!